client/state/preferences/test/actions.js

```diff
@@ -62,11 +62,19 @@ describe( 'actions', () => {
 			} );
 		} );
 
+		test( 'should dispatch PREFERENCES_RECEIVE when request completes', () => {
+			return fetchPreferences()( spy ).then( () => {
+				expect( spy ).to.have.been.calledWithMatch( {
+					type: PREFERENCES_RECEIVE,
+					values: responseShape[ USER_SETTING_KEY ],
+				} );
+			} );
+		} );
+
 		test( 'should dispatch success action when request completes', () => {
 			return fetchPreferences()( spy ).then( () => {
 				expect( spy ).to.have.been.calledWithMatch( {
 					type: PREFERENCES_FETCH_SUCCESS,
-					values: responseShape[ USER_SETTING_KEY ],
```

Comment:
Bug introduced in #7566.

Before that PR, `PREFERENCES_FETCH_SUCCESS` action included the settings. After that change, the settings are included in the new action `PREFERENCES_RECEIVE` instead.